Governance Polls and Atlas Edit 2024-11-18 (#2795)
makerdao/community: governance/polls/SparkLend Ethereum -  Adjust USDS DDM Parameters for Aave Lido Market - November 18, 2024.md

---
title: SparkLend Ethereum - Increase USDS DDM Maximum Debt Ceiling for Aave Lido Market - November 18, 2024
summary: Signal your support or opposition to increasing the USDS DDM Maximum Debt Ceiling for Aave Lido Market.
discussion_link: https://forum.sky.money/t/28-nov-2024-proposed-changes-to-spark-for-upcoming-spell/25543

>     input_format: single-choice

Reviewer: Good catch, thank you!

>     input_format: single-choice

Reviewer: ```suggestion
parameters:
    input_format: single-choice
```

    victory_conditions:
        - { type : plurality }
    result_display: single-vote-breakdown
version: v2.0.0
options:
   0: Abstain
   1: Yes
   2: No
start_date: 2024-11-18T16:00:00
end_date: 2024-11-21T16:00:00
---
# Poll: SparkLend Ethereum - Increase USDS DDM Maximum Debt Ceiling for Aave Lido Market - November 18, 2024

> The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Ecosystem team. This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9_Weekly_Governance_Cycle-95671ce7-60d8-4d8e-b9b6-de7493560308|0db3) will be active for three days beginning on Monday, November 18 at 16:00 UTC.

Reviewer: ```suggestion
The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Facilitator. This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9_Weekly_Governance_Cycle-95671ce7-60d8-4d8e-b9b6-de7493560308|0db3) will be active for three days beginning on Monday, November 18 at 16:00 UTC.
```

summary: Signal your support or opposition to increasing the USDS DDM Maximum Debt Ceiling for Aave Lido Market.
discussion_link: https://forum.sky.money/t/28-nov-2024-proposed-changes-to-spark-for-upcoming-spell/25543
    input_format: single-choice
    victory_conditions:
        - { type : plurality }
    result_display: single-vote-breakdown
version: v2.0.0
options:
   0: Abstain
   1: Yes
   2: No
start_date: 2024-11-18T16:00:00
end_date: 2024-11-21T16:00:00
---
# Poll: SparkLend Ethereum - Increase USDS DDM Maximum Debt Ceiling for Aave Lido Market - November 18, 2024

The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Ecosystem team. This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9_Weekly_Governance_Cycle-95671ce7-60d8-4d8e-b9b6-de7493560308|0db3) will be active for three days beginning on Monday, November 18 at 16:00 UTC.

**This is a binary vote.**

- **You may vote for a single option.**
- **You should vote for the option which you prefer.**
- **If you would accept either option, you should vote 'Abstain'.**

## Review

> The community can vote in this poll to express support or opposition to adjusting increasing the USDS DDM Maximum Debt Ceiling for the Aave Lido Market as described below:

Reviewer: ```suggestion
The community can vote in this poll to express support or opposition to increasing the USDS DDM Maximum Debt Ceiling for the Aave Lido Market as described below:
```
